ports/willyousnail/willyousnail/tools/patchscript

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# Set GAMEDIR to the current directory and set logfile
+GAMEDIR="$PWD"
+LOGFILE="$GAMEDIR/patchlog.txt"
+
+# Redirect output and error to the log file
+exec > >(tee -a "$LOGFILE") 2>&1
+echo "GAMEDIR is set to: $GAMEDIR"
+
+# Exports
+export DATADIR="$GAMEDIR/assets"
+export DATAFILE="willyousnail.port"
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/lib:$GAMEDIR/tools/libs:$LD_LIBRARY_PATH"
```

Comment:
You forgot to put the `libs` folder in `tools` (https://github.com/cdeletre/gmtools/tree/main/tools/libs). Audio compression will fail on some CFWs (it will produce empty sound entries).

Reply:
added. please check